db/migrate/20220608035549_remove_allocated_family_fee_from_nebraska_approval_amounts.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveAllocatedFamilyFeeFromNebraskaApprovalAmounts < ActiveRecord::Migration[6.1]
+  def up
+    remove_column :nebraska_approval_amounts, :allocated_family_fee
+  end
+
+  def down
+    add_column :nebraska_approval_amounts, :allocated_family_fee, :decimal
+  end
```

Comment:
```suggestion
  def change
    remove_column :nebraska_approval_amounts, :allocated_family_fee, :decimal
  end
```
if you add the column type, this should be reversible